src/config/services/user-services.md

```diff
@@ -13,7 +13,13 @@ following `run` script:
 ```
 #!/bin/sh
 
-exec chpst -u "<username>:$(id -Gn <username> | tr ' ' ':')" runsvdir /home/<username>/service
+export USER="<username>"
+export HOME="$(getent passwd "$USER" | cut -d: -f6)"
+
+groups="$(id -Gn "$USER" | tr ' ' ':')"
+svdir="$HOME/service"
```

Comment:
I feel like these are adding some unecessary indirection. Wouldn't the user who has a home directory with a different name from their username know what they need here? Exporting USER and HOME seems reasonable, though.

Reply:
```suggestion
export USER="<username>"
export HOME="/home/<username>"

groups="$(id -Gn "$USER" | tr ' ' ':')"
svdir="$HOME/service"
```